examples/src/main/scala/Protocol.scala

```diff
@@ -30,7 +30,8 @@ import codecs._
 @GenClient(remotely.Protocol.empty.codec[Int].specify1[Int,Int]("fac").signatures)
   object FacClient
 
-@GenServer(remotely.Protocol.empty.codec[List[Int]]) abstract class Foo
+// TODO(ahjohannessen): @stew <- Compiling Foo fails somehow?
+// @GenServer(remotely.Protocol.empty.codec[List[Int]]) abstract class Foo
```

Comment:
@timperrett the macro business should not be broken, I think it has to do with implict search failing. All tests are green in 2.10 and 2.11, btw. I just cannot figure out what goes wrong.